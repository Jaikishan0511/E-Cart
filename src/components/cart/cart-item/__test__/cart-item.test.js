import CartItem from "../index";
import React from "react";
import ReactTestRenderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
const CartItemComp = (
    <Router>
      <CartItem />
    </Router>
  );
describe("Actions", () => {
    describe("Render", () => {
      it("should be a valid React Element", () => {
        expect(React.isValidElement(<CartItem />)).toBeTruthy();
      });
  
      it("should render without throwing an error", () => {
        expect(() =>
          ReactTestRenderer.create(CartItemComp)
        ).not.toThrow();
      });
    });
  
    describe("Snapshots", () => {
      it("matches the default snapshot", () => {
        const tree = ReactTestRenderer.create(CartItemComp).toJSON();
        expect(tree).toMatchSnapshot();
      });
    });
  });