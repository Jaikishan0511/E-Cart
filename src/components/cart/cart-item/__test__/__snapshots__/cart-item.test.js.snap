// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actions Snapshots matches the default snapshot 1`] = `
<div
  className="cart-items"
>
  <div
    className="remove-icon"
    onClick={[Function]}
  >
    <img
      alt="logo"
      src="/images/close-icon.png"
    />
  </div>
  <div
    className="logo-box"
  >
    <img
      alt="logo"
      src="/images/logo.png"
    />
  </div>
  <div
    className="item-title"
  >
    
  </div>
  <div
    className="price-box"
  >
    Price Rs 
    0
  </div>
</div>
`;
